client/src/app/components/managing-body/managing-body.component.ts

```diff
@@ -0,0 +1,21 @@
+import {Component, Input, Output, EventEmitter} from '@angular/core';
```

Comment:
@tarieli1 Add one whitespace character inside the curly braces in the import (at the start and at the end) [Style 03-05]